site/docs/foundations/characteristics/index.mdx

```diff
@@ -0,0 +1,110 @@
+
+### Why characteristics?
+
+From a design perspective, there is a need for aesthetic flexibility without decreasing the efficiency of build or the effectiveness of the components when implemented. From a dev perspective, it provides an easy way to reuse the same values, without the overhead of having to remember details such as hex values. Any time you want to tweak the value of a characteristic, you only have to do it in one place for it to reflect everywhere.
+
+Characteristics offer the following benefits:
+
+- Ability to theme the Salt inline with third-parties or CIB sub brand guidelines.
+- Increase consistency across all components, with the knock-on effect of improving the overall user experience e.g. reduced cognitive load.
+- Simplified design of new components and patterns e.g. drag-and-drop behavior.
+- Streamlined evolution – a single change to a characteristic will automatically cascade down through all related components within the library.
+
+### Token name convention
+
+When defining characteristic design tokens, they should generally be in the format `--salt-<characteristic name>-<variant>-<scale>-<density>`
+
+1. Always begin with `--salt-`
+2. Follow this by the foundation name, e.g. `---salt-actionable-`, `---salt-text-`, `--salt-focused-`
+3. Append the variant if applicable, e.g. `--salt-actionable-cta-`, `--salt-text-h2-`
+4. Append the attribute, e.g. `--salt-actionable-cta-background`, `--salt-text-h2-fontSize`, `--salt-focused-outlineColor`
+5. Append the state if applicable, e.g. `--salt-actionable-cta-background-hover`
+
+### Styling with Characteristics
+
+Remember to wrap the characteristic you wish to use inside a `var()` function.
+
+For example, a Button is a component on which a user can perform an action, so you could use the following characteristics to style the component:
+
+```css
+.saltButton {
+  background: var(--salt-actionable-primary-background);
+  font-weight: var(--salt-actionable-fontWeight);
+}
+```
+
+Characteristics are both light/dark and density scoped, meaning that the value the characteristic points to will change according to the values given in the nearest `SaltProvider`. Considering the above, we would get that in light
+theme, `--salt-actionable-primary-background` resolves to `var(--salt-palette-interact-primary-background)`, which in light theme is `var(--salt-color-gray-60)` and in dark theme is `var(--salt-color-gray-300)`, of which foundation tokens then in turn resolve to their respective RGBA values.
+
+### Simple customization using characteristics
+
+In the following example, we will show how to customize the Card component through characteristics. Some components, like Button and Panel, have different variants such as 'cta' and 'secondary'. Currently, Card only has one variant, and uses tokens such as `--salt-container-primary-background` by default for all of its styling purposes. Here's how the Salt Card looks contained within a secondary Panel:
```

Comment:
"Here's how the Salt Card looks contained within a secondary Panel:"

needs image following